jenkins/pipeline.groovy

```diff
@@ -85,6 +85,21 @@ pipeline {
                 }
 
                 sh 'echo Prepare: \$(date -u "+%s")'
+                echo 'Checking Percona Server branch version, JEN-913 prevent wrong version run'
+                sh '''
+                    MY_BRANCH_BASE_MAJOR=5
+                    MY_BRANCH_BASE_MINOR=5
+                    RAW_VERSION_LINK=$(echo ${GIT_REPO%.git} | sed -e "s:github.com:raw.githubusercontent.com:g")
```

Comment:
why do we need to fetch it?
I think we can check version file content after `./local/checkout` command

Reply:
the idea is to terminate the job immediately and don't do checkout in case of version mismatch